README.md

```diff
@@ -0,0 +1,107 @@
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/user-attachments/assets/9335b488-ffcc-4157-8364-2370a0b70ad0">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/user-attachments/assets/3a7eeb08-1133-47f5-859c-fad4f5a6a013">
+  <img alt="nginx Banner">
+</picture>
+
+# Table of contents
+- [Overview](#overview)
+- [Documentation](#documentation)
+- [Installation](#installation)
+  - [Stable and mainline versions](#stable-and-mainline-versions)
+  - [Prebuilt binary package](#prebuilt-binary-package)
+  - [Building from source](#building-from-source)
+- [Windows](#windows)
+- [Support](#support)
+- [Contributing](#contributing)
+- [Security](#security)
+- [Changelog](#changelog)
+- [License](#license)
+
+# Overview
+nginx ("<i>engine x</i>") is an HTTP web server, reverse proxy,
+content cache, load balancer, TCP/UDP proxy server, and mail proxy server.
+Known for flexibility and high performance with low resource utilization.
+Originally written by [Igor Sysoev](http://sysoev.ru/en/).
+
+nginx runs on most popupar [operating systems](https://nginx.org/en/#tested_os_and_platforms).
+
+The official home page of the nginx project is [nginx.org](https://nginx.org).
+
+# Documentation
+Reference documentation is available at: https://nginx.org/en/docs.
+
+# Installation
```

Comment:
On second thought, given there is already a link to the docs above, I'd suggest to remove this and below sections all the way up to "Support".